src/actions/functionList.js

```diff
@@ -0,0 +1,56 @@
+//TODO: I don't like the filename... to many files with the same name
```

Comment:
again, I think the file name is fine. It makes sense and this comment could be removed